simpa_tests/automatic_tests/structure_tests/test_vesseltree.py

```diff
@@ -69,21 +69,22 @@ def test_bifurcation(self):
 
         assert has_split
 
-    def test_radius_variation_factor(self):
-        """
-        Test radius variation factor
-        Let there be no bifurcation or curvature, and see if the radius changes
-        :return: Assertion for radius variation
-        """
-        self.vesseltree_settings[Tags.STRUCTURE_RADIUS_VARIATION_FACTOR] = 1
-        ts = VesselStructure(self.global_settings, self.vesseltree_settings)
-
-        vessel_centre = 5
-        edge_of_vessel = vessel_centre + self.vesseltree_settings[Tags.STRUCTURE_RADIUS_MM]
-        has_reduced = np.min(ts.geometrical_volume[edge_of_vessel-1, :, vessel_centre]) == 0
-        has_increased = np.max(ts.geometrical_volume[edge_of_vessel+1, :, vessel_centre]) != 0
-
-        assert has_reduced or has_increased
+    # Isn't the radius only adjusted when there is a bifurcation happening?
+    # def test_radius_variation_factor(self):
+    #     """
+    #     Test radius variation factor
+    #     Let there be no bifurcation or curvature, and see if the radius changes
+    #     :return: Assertion for radius variation
+    #     """
+    #     self.vesseltree_settings[Tags.STRUCTURE_RADIUS_VARIATION_FACTOR] = 1
+    #     ts = VesselStructure(self.global_settings, self.vesseltree_settings)
+
+    #     vessel_centre = 5
+    #     edge_of_vessel = vessel_centre + self.vesseltree_settings[Tags.STRUCTURE_RADIUS_MM]
+    #     has_reduced = np.min(ts.geometrical_volume[edge_of_vessel-1, :, vessel_centre]) == 0
+    #     has_increased = np.max(ts.geometrical_volume[edge_of_vessel+1, :, vessel_centre]) != 0
+
+    #     assert has_reduced or has_increased
```

Comment:
I think that we should not remove the test but rather make it stable. For example by setting a seed, and disclosing in the documentation that setting `vesseltree_settings[Tags.STRUCTURE_RADIUS_VARIATION_FACTOR] = 1` might involve small variations due to rounding errors.

Reply:
A seed will not make the execution stable across OSs and machines, as the underlying PRNG might yield different number sequences. And I am not sure what you mean by the rounding errors. This sets the radius variation to +/- one pixel, but it is equally sampled between -1 and 1 -> depending on the discretization of the simulation grid, this might not result in a whole-pixel change of the vessel boundary.

Reply:
I agree that setting a seed would not fix it actually. 
And I understand what you mean now from this: 
https://github.com/IMSY-DKFZ/simpa/blob/develop/simpa/utils/libraries/structure_library/VesselStructure.py#L105
However, I would have imagined that when you set `vesseltree_settings[Tags.STRUCTURE_RADIUS_VARIATION_FACTOR] = 1` that this would translate to no variation, since it is a "factor" after all. And this is not the case because that line of code assumes that it is always different from 1. I think a better approach would be to set something like:
```python
if radius_variation == 1:
    radius_array = radius
else: 
    radius_array.append(np.random.uniform(-1, 1) * radius_variation + radius)
```

Reply:
At that point, the easier fix would probably be to rename the variable, such that it instead reads:
    
    Tags.STRUCTURE_MAX_VESSEL_RADIUS_VARIATION

?